charts/tfy-prometheus-config/templates/prometheusRules/kubecost-recording-rules.yaml

```diff
@@ -4,7 +4,7 @@ kind: PrometheusRule
 metadata:
   name: {{ .Values.prometheusRules.kubecostRules.name }}
   labels:
-    {{- include "prometheusRules.labels" . | nindent 4 }}
+    {{- include "recording-rules.labels" . | nindent 4 }}
```

Comment:
How having recording-rules.labels better then a single prometheus.Labels. We should add kubeCost labels also in the template function

Reply:
Please change the name here